packages/api/src/graphql/article.ts

```diff
@@ -284,7 +286,9 @@ export const GraphQLPublicArticle: GraphQLObjectType<
       })
     },
 
-    blocks: {type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLPublicBlock)))}
+    blocks: {type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLPublicBlock)))},
+
+    comments: {type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLPublicComment)))}
```

Comment:
Just to make sure that we got this right: do we need `GraphQLNonNull(GraphQLList` here? An article can also have 0 comments or do we just save an empty array `[]` if there are no comments? In that case your solution would work of course.

Reply:
When we work with lists/arrays you should always use NonNull.